core/src/operations/decorators/advice.rs

```diff
@@ -18,13 +18,19 @@ pub enum AdviceInjector {
     /// The result is injected into the advice tape as follows: first the remainder is injected,
     /// then the quotient is injected.
     DivResultU64,
+
+    /// Injects a vector of field elements at the front of the advice tape. The vector is taken
+    /// from tape map under a given key which corresponds to the top four elements on the stack.
+    /// The stack is expected to have at least 4 elements
```

Comment:
I'd change this to: "Injects a list of field elements at the front of the advice tape. The list is looked up in the key-value map maintained by the advice provider using the top 4 elements on the stack as the key.